callstack/react-native-builder-bob
# feat: use metro for web bundling in js library template

### Summary

This changes the bundler to Metro for the example app using Expo. It reduces the configuration required and aligns with the new defaults of Expo projects.

### Test plan

- Create a library and choose the "JavaScript library" example
- Ensure that the project runs on the web

<img width="995" alt="image" src="https://github.com/callstack/react-native-builder-bob/assets/1174278/8dbe93dd-730f-49cd-9147-a7ec6f56e140">
